src/main/java/org/openstreetmap/atlas/utilities/runtime/RunScript.java

```diff
@@ -20,20 +20,20 @@
 {
     private static final Logger logger = LoggerFactory.getLogger(RunScript.class);
 
-    public static void run(final String script)
+    public static void run(final String[] commandArray)
```

Comment:
Please keep the single String option as well, so as to make it non breaking.

Reply:
done